device/rsvp/rsvpIngressP2PLsp.yaml

```diff
@@ -0,0 +1,91 @@
+components:
+  schemas:
+    Rsvp.LspInterface.IngressP2PLSP:
+      description: >-
+        Configuration for one RSVP Ingress point-to-point LSP.
+      type: object
+      required: [remote_address]
+      properties:
+        remote_address:
+          description: >-
+            IPv4 address of the remote endpoint of the LSP.
+          type: string
+          format: ipv4
+        tunnel_id:
+          description: >-
+            The Tunnel ID of the RSVP LSP. Carried in the SESSION object in Path Messages.
+          type: integer
+          default: 1
+          minimum: 1
+          maximum: 65535
+        lsp_id:
+          description: >-
+            The LSP ID of the RSVP LSP. Carried in the SENDER_TEMPLATE object in Path Messages.
+          type: integer
+          default: 1
+          minimum: 1
+          maximum: 65535
+        refresh_interval:
+          description: >-
+             The refresh interval in seconds after which refresh Path Messages are periodically sent. 
```

Comment:
The refresh interval in seconds. It is the time interval between periodic refresh Path Messages.

Reply:
Done.